test/evergreen/code_complexity_analysis.py

```diff
@@ -0,0 +1,126 @@
+import argparse
+import ast
+from collections import defaultdict
+import csv
+from heapq import nlargest
+import json
+import os.path
+import pandas as pd
+
+def get_atlas_compatible_code_statistics(summaryFile, dataFile, outfile='atlas_code_complexity.json'):
+    ''' Generate the Atlas compatible format report. '''
```

Comment:
Suggest adding a blank line after the docstring (for ease of reading). This is a general comment applied to the below functions too.

Reply:
Fixed in 3154760ada6586d59d072c5df2e956a40df9eba9